src/genesys/data.py

```diff
@@ -72,6 +72,7 @@ def _add_column(dataset, path):
 
         self.total_samples = min(max_samples, total_samples)
 
+        self.tokenize = tokenize
         self.tokenizer = tokenizer
```

Comment:
lets differentiate both variable. tokenize and tokenizer are difficult to put apart

Reply:
changed to `do_tokenization`